src/routes/ai.tsx

```
@@ -0,0 +1,140 @@
+      {
+        name: "description",
+        content:
+          "Connect explicitly delegated Gapwise timetable and UTM campus intelligence to compatible AI assistants through Gapwise's secure remote MCP service.",
+      },
+    ],
+  }),
+  component: AiPage,
+});
+
+const cardClass =
+  "rounded-2xl border border-border bg-card/40 p-5";
```

ESLint

[error] Delete `⏎·`

(prettier/prettier)